rabbitmq/files/entrypoint.sh

```diff
@@ -1,6 +1,29 @@
-{%- from "rabbitmq/map.jinja" import server with context -%}
+{%- from "rabbitmq/map.jinja" import server,cluster with context -%}
 #!/bin/bash -e
 
+{%- if cluster.enabled %}
+
+# if NUM_NODES not passed, default to 3
+if [ -z "$NUM_NODES" ]; then
+  NUM_NODES=3
+fi
+
+# loop through number of nodes
+for NUM in `seq 1 $NUM_NODES`; do
```

Comment:
1. use lower-case for local variables during iteration
2. don't need to exec seq here
`for num in 1..${NUM_NODES}`